functionality.md

```diff
@@ -84,16 +79,10 @@ Now that you have tested your application's basic functionality, you can add som
 
 ## Step 5 — Adding Validations
 
-Your book application can accept input from users, but imagine a case where a user attempts to create a book without adding a title to it, or creates an entry for a book that's already in the database. You can create mechanisms to check data before it gets entered into the database by adding validations to your models. Since your application's logic is located in its models, validating data input here makes more sense than doing so elsewhere in the application.
+Your book application can accept input from users, but imagine a case where a user attempts to create a book without adding a title to it, or creates an entry for a book that's already in the database. You can create mechanisms to check data before it gets entered into the database by adding validations to your models. Since your application's logic is located in its models, validating data input here makes more sense than doing so elsewhere in the application. Rails provides [lots of great validation options](https://guides.rubyonrails.org/active_record_validations.html) for us to use.
 
 Note that we will not cover writing validation tests in this tutorial, but you can find out more about testing by consulting [the Rails documentation](https://guides.rubyonrails.org/testing.html).
 
-If you haven't stopped the server yet, go ahead and do that by typing (in separate terminal)
-```
-dip down
-```
-or press `CTRL + C`
```

Comment:
I like having this in, maybe not here. But it might be good practice for someone to learn to start/stop a couple times. If this exists somewhere else with both of these versions, then we probably don't need it.

Reply:
Understood. But I don't want someone thinking "I have to restart my server after I make a change to the view, or controller, or..." I want them to be knowledgeable enough to know that they only have to restart it during a finite set of circumstances... or if something goes wrong.